src/main/java/org/session4/homework/Temperature.java

```diff
@@ -0,0 +1,18 @@
+package org.session4.homework;
+
+public class Temperature {
+
+    public String perceivedTemperatureEffect(int temperatureCelsius ) {
+        if (temperatureCelsius > 30){
+            return "Hot";
+        } else if (temperatureCelsius <= 30 && temperatureCelsius >= 20) {
+            return "Warm";
+        } else {
+            return "Cold";
+        }
+    }
+
+    public void displayPerceivedTemperatureEffect(String s){
```

Comment:
Please don't use one-letter parameters.
Also, on the backend side, a more intuitive action would be `print` or `log`, because we can relate these actions to the print method from Java, or log from logging libraries. There is nothing wrong with `displayPerceivedTemperatureEffect`, but is a suggestion.

Reply:
ok, I was a bit confused, since last week you mentioned something about the display word also, but I agree that print/log sounds better